scale-testing/cluster-density/templates/secret.yml

```diff
@@ -0,0 +1,7 @@
OFFwWjlaalN3cEVVQ3htekVKCjArRWIzUWFBQzVWSTFCYlVzc0VZWjZWb0ZJaEEweHp1ZHJiRG1SWkJ3T2Exc3VmUDA2ZU9MQWZJZEJKNmZ5d0ZHQjdDR0IKWkNMcmpZSDM3R0FIS2s0VlFRRGVVcjJUaGI5MFA2R0N4V1dhSkh5clU3dUFxVjQxMEhOOEJWMFRjUmtjZys5djdxdTJzbwpQaVhZYXNCY0RSa2E3VE1hNk1qdU92bDJYRjBVcWJVNE9Qai81RTBzZTMrRGdkK0JzM3BHbzBjUGU4QnFkUEZ0bTNsMmRjClNzUzMvaVRyS2VBcitEMWNobXpZVExiNXREOERPMU5xZzJ1TU1ESmxBQUFGbUFSV2hLa0VWb1NwQUFBQUIzTnphQzF5YzIKRUFBQUdCQU52a3RSc3FPQ0xEZjZPVGZ4UUE5clhpaUtIa2JpdXUvNUtxMFBodmNOR0lzVktscU4zK3FBbFFTck1jdWZXYwpIMXBzb1ExbFV1NWdPVTkveEJGcFM1QmFORW55YVo1WmlxV1k0SC9HRUUwRVNqMzBCM3dsWkVQVlQrQWdGQlRTSW5vY0FzClB6VmFEU0dhSmlpYUU1Y05jQlY1NDl1MFVIRWc5MEIyL2RGUXBRSUZ5bGdCK3lIYjFPeTk1SVhRZlViYVFPODRsdE1FYVoKWWlERThHekJIaXdEYlVHNkgxQS9hTGpTVmNsUU4yOFN5K1h2MTJOdWV2RUtXZldZMHNLUkZBc1pzeENkUGhHOTBHZ0F1VgpTTlFXMUxMQkdHZWxhQlNJUU5NYzduYTJ3NWtXUWNEbXRiTG56OU9uaml3SHlIUVNlbjhzQlJnZXdoZ1dRaTY0MkI5K3hnCkJ5cE9GVUVBM2xLOWs0Vy9kRCtoZ3NWbG1pUjhxMU83Z0tsZU5kQnpmQVZkRTNFWkhJUHZiKzZydHJLRDRsMkdyQVhBMFoKR3Uwekd1akk3anI1ZGx4ZEZLbTFPRGo0LytSTkxIdC9nNEhmZ2JONlJxTkhEM3ZBYW5UeGJadDVkblhFckV0LzRrNnluZwpLL2c5WElaczJFeTIrYlEvQXp0VGFvTnJqREF5WlFBQUFBTUJBQUVBQUFHQkFNTnUvc09ZLzNGdVBkZ2RORm1NRGJsVUVFCm1LbzV3eG1iWTZHU1prcldtZFNyU0F4WmdnU0NxVzhQMUY3OFRxTURlR3lwRUVWL1BERTh5amFwaHE5QnprL3dvbDZGY3oKai9WajdVdkN0aVF0UnVyUEk0UGtHN3ZxTDJhc1BBelJRcEV5cWdhTUU3T3VaOWdhS2VaZ0Rma1ptVXpyNHJBVCs2bytYbwptK2ZYKy8vVE02YUtoQy93bU1xY2J5bTdSa3lkbUE3RlJjOWdZWk5zeVhISFFjSTN2TXcrTWh6OGpid25CWmlLTTJwejRSCi82UnFTUTEzV004NGV3RzhCTDJtZi9vRDJDd0pYU1Izb2UzT1NOWTlzMWs5NlhTbjJSem5Db0dORDRjZ1B6dzZiNS96TmYKKzRGU1RpUXA1OWJJM1A3QjVuMHVSdFVleml6V0xtQ00zemdsQlF2S1lQN0FIWmcra2IrY1NySU5nM0FlTVd0WklDUHdVUQovOXRzZ1ora0g5dnFvT3NGeXV1OVZnYlpBdDNRbXFKM2VxRFR2THlkMHprVTZrcHR3YTdnNWpvMmJpT1ZnN1QzZTFBV3JxCjM1cWRLcmpXMlBGVWpnZVJ6QiszVVdjVkozMS9xWTNENzQwWU4raFBpQTV6L3Jrb3dSdEc5RFVmMmtneXhvZ2FpOEFRQUEKQU1CV3FNbW9ERUhRbnlOdnlBT3lVellHV3NjelpvNm4vVTQvTkNvNlAzTzZCYnRWS0FUV1ZYWEVtNjVnZTBZTVRQYlJqWgpkWnE3d2hPV0h6cjVKRTNGanl4RllJWWEvdThPcVdjOEtzeUlIakZzM1dBT0t0RWI0L0x2WjFzWk5KT1I3OVVBYStud3ZqCm1rQXp6d3FxUTBzSFV0UWVoVHV0OTFNaURFNVB6OW8wdFZnQ25FbmJ1eGNuOEdRSmZONW1JZ0E2YnNzRDdmRWo3Sk03dnIKSE5ETjYzUjd2UTUyNTNNQ3lpUG93NW9iV01PcWZMU3VJVGxKNjg3NWJHQWNLQ2dNSUFBQURCQVA5M0l5MVFXcUptWnhsSQpueUZlU3g4Nk5meEJKV3hHc2l1MndqOE12ZldyWXQ4V2pQNHZjSEtSNjcydlc5UEs0Qk1GcDVNSVl2YkFnbGxhYnp2VVVyClF1VGhITWI3czlUL0JHM09yQUpyemVNZURrK3lybzlJaC9jRmxGS01xUGRoZzNLVEYxU1Jzb2F5L2pRVkh5QVIvY0ZnUmUKNzVaM3NQczNGOFJiSlpzL3VNUUxFbUJHVXAvNG9YbG1kKyt1c1FET2ptUWxsYm44bHh1N3gxbU42MDNzKytTVWVxWVVKZQo4cng5Y2hEU2tFTTJQWEhpdm9zbVUzdUZJaTRteUJBUUFBQU1FQTNGcURSQWliRVB3TGRqOWQ1UjJoTjI0eklmSGtadkEvClJxaDhzOCt5ZFUzakJQRGdGRUQwSGl4YXl0MVhnTzEwZDBGZEEvSFI2cE8xdUdybDdzdm5nYjdnSGJpTGo5SnBvRGd0SWkKalZnSEZvcmFZMWpTeGEyemNNbldFWVFaVFRlYUF6TU5qdWlJb2MwemN3ZHBRTWpkWWdwOUJBK1pVVEgvZTJNVmpmdUtmQQo3SlRDSW1uQUJlUWV1SWRtZ1VTZkltUTdRNGJsL29uUUZUZkVLc0tpZVFCU3hjS1VLa0orZ3R2RTJ5ZWJRRVFMV09kaE5GCjJ0S05TQlk5TUFsRTFsQUFBQUlISnpaWFpwYkd4aFFIZHZibVJsY214aGJtUXVjbk5sZG1sc2JHRXViM0puQVFJPQotLS0tLUVORCBPUEVOU1NIIFBSSVZBVEUgS0VZLS0tLS0K
```

Comment:
did you mean to commit this?

Reply:
I did, this comes from kube-burner examples, I believe it's just a generic secret to add to the projects kube-burner creates. We should probably replace it with a generic secret though as I can generate a valid public key from the decoded private key.

Reply:
I've edited the key to make it invalid and re-encoded it.